_templates/post

```diff
@@ -5,10 +5,11 @@ modified:
 categories: {{ dir }}
 description:
 tags: []
+time: XX minutes
```

Comment:
hi @ccjjmartin, thanks for this PR. I'd prefer to have the time empty by default. Can you update this line for ``time:``